identicon.js

```diff
@@ -33,11 +33,19 @@
         if (typeof(arguments[1]) === 'number') { this.options.size   = arguments[1]; }
         if (arguments[2])                      { this.options.margin = arguments[2]; }
 
-        this.hash        = hash                    || this.defaults.hash;
+        this.hash        = hash                    || this.defaultHash();
         this.background  = this.options.background || this.defaults.background;
         this.margin      = this.options.margin     || this.defaults.margin;
         this.size        = this.options.size       || this.defaults.size;
         this.format      = this.options.format     || this.defaults.format;
+        this.numCells    = this.options.numCells   || this.defaults.numCells;
```

Comment:
I don't think `numCells` is an accurate name for what this variable represents. Is it perhaps `squareSize`?